.github/workflows/ci.yml

```diff
@@ -79,23 +79,3 @@ jobs:
         run: sudo xcode-select -s /Applications/Xcode_15.4.app
       - name: Build ${{ matrix.config }}
         run: make CONFIG=${{ matrix.config }} build-swift-apple-platforms
-      # - name: Install Docker
-      #   run: |
-      #     HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask docker
-      #     sudo /Applications/Docker.app/Contents/MacOS/Docker --unattended --install-privileged-components
-      #     open -a /Applications/Docker.app --args --unattended --accept-license
-      #     echo "We are waiting for Docker to be up and running. It can take over 2 minutes..."
-      #     while ! /Applications/Docker.app/Contents/Resources/bin/docker info &>/dev/null; do sleep 1; done
-      # - name: Start test infrastructure
-      #   run: docker compose up --debug
-      #   working-directory: test/scripts/forked_state
-      # - name: Wait for local RPC to be ready
-      #   run: while ! curl localhost:8545/health; do sleep 1; done
-      # - name: Wait for local bundler to be ready
-      #   run: while ! curl localhost:4337/health; do sleep 1; done
-      # - name: Wait for local paymaster to be ready
-      #   run: while ! curl localhost:3000/ping; do sleep 1; done
-      # - name: Run ${{ matrix.config }} tests
-      #   run: make CONFIG=${{ matrix.config }} test-swift-apple-platforms
-      - run: git diff crates/ffi
```

Comment:
Can we restore these two git commands? I think they need to go just after the `make build-xcframework` command. This will check if this command writes to bindings files (meaning we need to run this command locally in order to commit the changes)

Reply:
crates/ffi its the old ffi files dir